Formula/rabbitmq-c.rb

```diff
@@ -21,7 +22,7 @@ class RabbitmqC < Formula
   def install
     system "cmake", ".", *std_cmake_args, "-DBUILD_EXAMPLES=OFF",
                          "-DBUILD_TESTS=OFF", "-DBUILD_API_DOCS=OFF",
-                         "-DBUILD_TOOLS=ON"
+                         "-DBUILD_TOOLS=ON", "-DCMAKE_INSTALL_RPATH=#{lib}"
```

Comment:
```suggestion
                         "-DBUILD_TOOLS=ON", "-DCMAKE_INSTALL_RPATH=#{opt_lib}"
```
Probably doesn't matter, but I suspect it's safer to use a path that persists across version bumps.

Reply:
Changed to `opt_lib`